guide/daily-spending-wallet/landing-page.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: guide
+title: Daily spending wallet
+description: Learn fundamentals, techniques and best practices for designing great bitcoin UX.
+nav_order: 4
+has_children: true
+permalink: /guide/daily-spending-wallet/landing-page/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/case-studies/cloud-backup/daily-spending-preview.jpg
+---
+
+<!--
+
+Illustration source:
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/cloud-backup/daily-spending-header.jpg"
+   retina = "/assets/images/guide/case-studies/cloud-backup/daily-spending-header@2x.jpg"
+   mobile = "/assets/images/guide/case-studies/cloud-backup/daily-spending-header-mobile.jpg"
+   mobileRetina = "/assets/images/guide/case-studies/cloud-backup/daily-spending-header-mobile@2x.jpg"
+   alt-text = "Various screens from a mobile bitcoin wallet"
+   caption = ""
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Daily spending wallet
+
+This section is an in-depth design exploration of a bitcoin-only, non-custodial, mobile lightning-first, wallet for on-the-go payments.
+
+Imagine a product which tries to solve the problem of quickly and easily sending smaller amounts of money to friends and family, or paying for [small purchases]({{ '/guide/designing-products/personal-finance/#day-to-day-spending' | relative_url }}). Ease and speed of use will be important as usage is likely to be on mobile devices and on the go. Users are not expected to be well-versed in bitcoin technology or advanced private key management, making it reasonable to worry more about self-inflicted loss than theft.
+
+---
+
+### [First use]({{ '/guide/daily-spending-wallet/first-use/' | relative_url }})
+
+An overview of common tasks when using a wallet application for the first time.

```

Comment:
We should rewrite these short descriptions to avoid repeat of "An overview" and "An introduction". Can be a separate PR.